docs/templates/about.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <script src="https://kit.fontawesome.com/3f00b2ab29.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="../styles/style.css">
+    <link rel="stylesheet" href="../styles/about-style.css">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About - Get Inspired</title>
+</head>
+<body class="no-margin">
+    <!-- Section 1 -->
+    <section class="section-1">
+        <div class="menu-bar"> 
+            <span class="a-span">
+                <i id="menuBtn" class="fa-solid fa-bars fa-2x icon-list"></i>
+              </span>
+        </div>
+        <nav class="nav-bar">
+            <div>
+                <i class="fa-brands fa-facebook-f"></i>
+                <i class="fa-brands fa-twitter"></i>
+                <a href="#">English</a>
+                <a href="#">My page</a>
+                <a href="#">Logout</a>
+            </div>
+        </nav>
+        <nav class="nav-bar nav-bar-2 drop-shadow">
+            <div>
+                <a href="/get-inspired/index.html"><img src="../sprites/22-removebg-preview.png" alt="Get inspired logo"></a>
+            </div>
+            <div class="no-margin">
+                <a  href="/get-inspired/templates/about.html">About</a>
```

Comment:
- [ ] Kindly change this link to index.html, this way users can easily navigate back to the home page from the about page 👍🏿

Reply:
I must have the repo on the path of the href as you see because that's how I make possible going to home page from about page. If I change this, the deployment would rise to an 404 error.